Dockerfile.invocation-image

```diff
@@ -1,8 +1,8 @@
 ARG ALPINE_VERSION=3.10.1
 
-FROM dockercore/golang-cross:1.12.9@sha256:3ea9dcef4dd2c46d80445c0b22d6177817f4cfce22c523cc12a5a1091cb37705 AS build
+FROM golang:1.12.9 AS build
```

Comment:
As we want ot move to Golang 1.13, maybe you can bump version at same time ?

Reply:
I did a force-push with golang:1.13.0. Built on amd64 and arm to verify. 

I'm guessing you guys already have the resources, but if you need access to aarch64 I'm sure you could get these guys to help you out: https://www.worksonarm.com/cluster/

The server they gave me was a Cavium which can't do armhf, but I've had good luck using the new AWS aarch64 instances and then running dockerd inside a 32bit LXD instance.